libraries/3rdparty/mbedtls/include/mbedtls/config.h

```diff
@@ -29,6 +29,9 @@
 #ifndef MBEDTLS_CONFIG_H
 #define MBEDTLS_CONFIG_H
 
+/* Standard include. */
```

Comment:
Ideally, the only changes we should be making to the mbedTLS configuration file are when we comment and un-comment #define lines in order to deviate from their default feature settings. Anything beyond that adds to the merge complexity that already deal with when mbed adds features and/or changes defaults.